console/src/components/Services/Data/TableInsertItem/InsertActions.js

```diff
@@ -40,8 +40,7 @@ const insertItem = (tableName, colValues) => {
       const colSchema = columns.find(x => x.column_name === colName);
       const colType = colSchema.data_type;
       if (Integers.indexOf(colType) > 0) {
```

Comment:
Can remove this check entirely because this is what happens in the final `else`.

Reply:
Done